data/cpu-x.1

```diff
@@ -0,0 +1,85 @@
+.TH CPU-X 1 "Oct 01, 2019"
+.SH NAME
+cpu-x \- X11 Tool that gathers Information on CPU, Motherboard and more...
+.SH SYNOPSIS
+.B cpu\-x
+[
+.I <DISPLAY>
+]
+[
+.I <OPTIONS>
+]
+.B cpu\-x_polkit
+[
+.I <DISPLAY>
+]
+[
+.I <OPTIONS>
+]
+
+.SH DESCRIPTION
+.B CPU\-X
+is similar to CPU\-Z (for MS Windows). It can be used in graphical
+mode by using GTK or in text\-based mode by using NCurses. A dump mode is
+present from the command line.
+.PP
+Parts of
+.B CPU\-X
+require root privileges. If your desktop session has PolicyKit support,
+you can launch \fIcpu\-x_polkit\fR. After admin authentication,
+\fBCPU\-X\fR will present to you its full feature set.
```

Comment:
Note, that this man page has been written for the 3.2.4 release. I saw that the cpu-x-root.desktop.in file has vanished on master, so maybe the above passage needs to be rewritten or dropped. Please adapt to the code situation on master as needed after applying this PR (I am not available for fine-tuning this).